README.md

```diff
@@ -66,15 +66,20 @@ New methods are added to `Map.prototype`.
   * `Map.prototype.filter(predicate, thisArg)`
   * `Map.prototype.mapValues(fn, thisArg)`
   * `Map.prototype.mapKeys(fn, thisArg)`
-
 * New methods:
+  * `Map.prototype.deleteAll(...elements)` - similar to `Set.prototype.deleteAll`.
```

Comment:
should there be a `setAll`, then?

Reply:
Something like that. `Set.prototype.set` accept 2 arguments, so it will not work by the same way as `.addAll` or `.deleteAll`.

Reply:
i'd expect `setAll` on Maps to accept entries as arguments.